action.yml

```diff
@@ -34,53 +34,74 @@ runs:
       shell: bash
       env:
         TYPE: ${{ steps.gettype.outputs.type }}
+        GITHUB_REPOSITORY: ${{ github.repository }}
       run: |
         major_type=$(echo $TYPE | cut -d "," -f 1)
         minor_type=$(echo $TYPE | cut -d "," -f 2)
-        # https://docs.github.com/en/rest/branches/branches#list-branches
```

Comment:
Why remove this? It's nice to have a quick reference to the relevant docs for API calls. In gha-ci we explicitly have them for each API call. Would be good to add one for the tags call below, too.

Reply:
Updated